tests/marine/test_marine.py

```diff
@@ -326,6 +326,43 @@ def test_filter_and_parse_without_fields(
     )
 
 
+def test_preferences_change_effect_while_running(
```

Comment:
I really like this test, nice!

If you decide to add the getter function to prefs, also assert that the tcp checksum verification is actually 0.